src/drone.py

```diff
@@ -0,0 +1,171 @@
+import roslibpy
+from abc import ABC, abstractmethod
+from enum import Enum
+
+class Drone(ABC):
+
+    class Flight_Status(Enum):
+        ON_GROUND_STANDBY = 1
+        IN_AIR_STANDBY = 2
+        FLYING = 3
+        FLYING_HOME = 4
+        PAUSED_IN_AIR = 5
+        LANDING = 6
+        NULL = 7
+
+    class UpdateMissionAction(Enum):
+        CONTINUE_MISSION = 0,
+        UPDATE_CURRENT_MISSION = 1,
+        END_AND_HOVER = 2
+
+    class WaypointActions(Enum):
+        START = 0
+        STOP = 1
+        PAUSE = 2
+        RESUME = 3
+
+    speed = 5
```

Comment:
Could we get a comment explaining what this particular speed variable is for? I don't think it's used anywhere else in the file.

Reply:
Fixed.  Supposed to be the speed of the drone in flight.  Set_speed should update this.